src/utils/registration-helpers.ts

```diff
@@ -698,4 +700,29 @@ export const willResetNviStatuses = (persistedRegistration: Registration, update
   if (hasChangedCategory) {
     return true;
   }
+
+  const persistedRegistrationWithContextId = persistedRegistration as JournalRegistration | ChapterRegistration;
+  const updatedRegistrationWithContextId = updatedRegistration as JournalRegistration | ChapterRegistration;
+  const hasChangedJournalId =
```

Comment:
husk å sjekk om endring av variabelnavn også endrer andre navn
```suggestion
  const hasChangedContextId =
```